docs/guide.md

```diff
@@ -9,75 +9,74 @@ description of LimeIDL please refer to the [LimeIDL description](lime_idl.md).
 Overview
 --------
 
-Gluecodium generates a C++ API with language bindings for Java and Swift. For C++ it will
+Gluecodium generates a C++ API with language bindings for Java, Swift, or Dart. For C++ it will
 generate declarations only. These need to be implemented by you to provide application logic.
-Java and Swift bindings are completely generated and forward all calls to C++.
+Java, Swift, and Dart bindings are completely generated and forward all calls to C++.
 
 Crossing the language boundary
 ------------------------------
 
-When building a Swift or Java app, the entry point for the program will be in Java/Swift.
-Calls then can be made from Java/Swift into C++ code. Java/Swift can call static or
-non-static member functions. Calling into Java/Swift from C++ is only possible on objects
-passed from Java/Swift to C++ first.
+When building a mobile app, the entry point for the program will be in Java/Swift/Dart (referred
+further as "platform code"). Calls then can be made from platform code into C++ code. Platform code
+can call static or non-static member functions. Calling into platform code from C++ is only possible
+on objects passed from platform code to C++ first.
 
-**Note:** In Java the native library needs to be loaded manually before any call to C++ is
-possible.
+**Note:** Both in Java and in Dart the native library needs to be loaded manually before any call to C++ is possible.
 
 Classes
 -------
 
 Classes are generated as an abstract class in C++ and as wrapper classes containing all the
-conversion and glue code in Java/Swift. These classes have their logic always on C++ side.
+conversion and glue code in platform code. These classes have their logic always on C++ side.
 New instances can only be created if it is the return value of a function and it is up to
 the users C++ implementation which subclass is returned. When the function is called from
-Java/Swift the instance will be automatically wrapped.
+platform code the instance will be automatically wrapped.
 
-**Note:** There is no pointer equality on Java and Swift side for the same wrapped C++ object.
+**Note:** As of now, there is no guarantee of pointer equality on platform side for the same wrapped C++ object.
```

Comment:
It would be nice to provide examples when it's guaranteed and when - not

Reply:
Done